src/Gitonomy/Git/Hooks.php

```diff
@@ -14,6 +14,7 @@
 
 use Gitonomy\Git\Exception\InvalidArgumentException;
 use Gitonomy\Git\Exception\LogicException;
+use RuntimeException;
```

Comment:
I would use this one https://github.com/gitonomy/gitlib/blob/1.3/src/Gitonomy/Git/Exception/RuntimeException.php insteand

Reply:
Oh yeah, that was an oversight, thanks!